types/audiosprite/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for audiosprite 0.6
+// Project: https://github.com/tonistiigi/audiosprite
+// Definitions by: Gyusun Yeom <https://github.com/Perlmint>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace audiosprite;
+export = audiosprite;
+
+declare function audiosprite(files: string[], option: audiosprite.Option, callback: (error: Error, obj: audiosprite.Result) => void): void;
+
+declare namespace audiosprite {
+    type ExportType = "jukebox" | "howler" | "createjs";
```

Comment:
`jukebox` is from cli help in README. It looks like default format name is `jukebox`, but not specified in API.